### ISSUE TITLE
werkt niet 

### ISSUE THREAD

**Issue by thomas192000** [robertklep/homebridge-nefit-easy#23]
ik kan geen temperatuur laden of instellen met de woning. app van apple en ik kon dit voor de update wel ik heb een junker thermostaat

**Comment by robertklep** [comment]
Over welke update heb je het?

**Comment by thomas192000** [comment]
Beste 

Ik had nefit -easy geinsttaleer op mijn homebridge maar die was gecrasht dus heb ik deze op nieuw moeten in stellen maar nu krijg ik nefit-easy niet meer aan de praat de thermostaat komt zichtbaar in de woning app maar ik kan geen temperatuur aflezen of in stellen en krijg de melding van apparaat reageert niet 

Ik heb een junker thermostaat 

Dit was de vorige update daar werkte het nog wel 




met vriendelijke groeten 
Thomas Boutsen

**Comment by robertklep** [comment]
Zou je de command line tool ([`nefit-easy-cli`](https://github.com/robertklep/nefit-easy-cli)) eens kunnen installeren en uitproberen?

**Comment by thomas192000** [comment]
Deze heb ik ook al geïnstalleerd  op mijn raspberry maar ik weet niet waar ik deze moet invoegen op mijn config.json voor de homebridge





met vriendelijke groeten 
Thomas Boutsen

**Comment by robertklep** [comment]
De commandline tool staat los van Homebridge, je kunt 'em gewoon uitvoeren als je op de Raspberry Pi bent ingelogd:
```
easy --serial=SERIAL --access-key=KEY --password=PASSWORD status
```

Je moet `SERIAL`, `KEY` en `PASSWORD` vervangen voor de juiste waarden.

**Comment by tliet** [comment]
Hi Robert, 

Dank voor het ontwikkelen van deze tools! Ik heb het 2 jaar geleden aan de gang gehad, maar na een paar maanden maakte ik het zelf stuk door te goochelen met node versies en kennis gebrek in het algemeen :-) Getracht met het bestaande image een en ander te repareren, maar uiteindelijk opgegeven en opnieuw begonnen.

Nu heb ik een compleet nieuwe Raspbian Lite geïnstalleerd, node 12[laatste] erop, homebridge geïnstalleerd en gekoppeld aan mijn iOS device maar de Nefit plugin lijkt het bij mij hetzelfde te doen als de persoon hier boven. Niets dus. 'Thermostaat' doesn't respond. Ik heb je laatste suggestie ook geprobeerd maar de easy cli tool blijft gewoon 'hangen'. De configuratie heb ik overgenomen van het oude OS, in ieder geval zijn accesskey en wachtwoord naar mijn weten niet gewijzigd.

Ik kan pingen naar buiten, DNS werkt, er zit geen filter op de verbinding. Kortom; ik ben nu ook op zoek naar wat richting.

Alvast bedankt voor je antwoord!

**Comment by robertklep** [comment]
@tliet als de CLI tool blijft hangen moet je even hier naar kijken: https://github.com/robertklep/nefit-easy-http-server#problems-on-recent-linux-distributions

**Comment by tliet** [comment]
@robertklep Top! Dat was de oplossing, nu werkt het weer! Mijn dank is groot.

**Comment by KoenVoortman** [comment]
Super!! Dank voor deze tip! Werkt nu weer als tevoren. Hier ook foutmelding, updates gedaan, uiteindelijk nieuwe installatie (Buster) en nieuwste node.js maar werkte niet... SSL settings aangepast en klaar!! 
Bij installatie geeft hij trouwens wel een melding over XMPP die verouderd is... is daar nog iets anders aan te doen?

**Comment by robertklep** [comment]
Nee, daar kun je niks aan doen. Ik heb (ooit, lang geleden) de nieuwe XMPP library gebruikt maar dat leverde een hele hoop problemen op. Die zullen vast allemaal wel opgelost zijn, maar sindsdien gebruik ik weer de oude (want dat werkt in ieder geval ;).

**Comment by jacobsigor** [comment]
Beste Robert, Ik zag de oplossing die je hier omschrijft. Ik ben alleen nieuw in het hele systeem en geen idee hoe ik deze aanpassing moet doen. Is er ergens een stappenplan voor de fix? Want ik weet nog niet hoe ik een file moet bewerken op mijn rasberry pi met Homebridge.

**Comment by robertklep** [comment]
@jacobsigor nee, er is geen stappenplan voor die fix, in ieder geval niet van mij, omdat het een probleem is binnen de Linux distributie en niet per se van deze plugin. Uiteindelijk moet je dus een systeembestand aanpassen, misschien dat je ergens kunt vinden hoe dat normaal gesproken in z'n werk gaat op Linux.

**Comment by jacobsigor** [comment]
dank voor je feedback. Ga ik even verder zoeken
